src/apps/reservation-list/modal/pause-reservation/pause-reservation.tsx

```diff
@@ -31,41 +31,45 @@ const PauseReservation: FC<PauseReservationProps> = ({ id, user }) => {
   );
   const [endDate, setEndDate] = useState<string>("");
 
-  const save = useCallback(() => {
-    if (user) {
-      // TODO: consolidate with the other save patron function
-      // be aware the defaults are not necessarily the same in the different save patron functions
-      const saveData = {
-        preferredPickupBranch: user.preferredPickupBranch,
-        receiveEmail: user.receiveEmail,
-        receivePostalMail: user.receivePostalMail,
-        receiveSms: user.receiveSms
-      } as Patron;
+  const save = useCallback(
+    (e: FormEvent<HTMLFormElement>) => {
+      e.preventDefault();
+      if (user) {
```

Comment:
I think it is prettier to make a guard, which also reduce indentation - in this case for quite a big block of code.
So instead of checking for user is true you can return quickly in the beginning if it is false.
```typescript 
  const save = useCallback(
    (e: FormEvent<HTMLFormElement>) => {
      e.preventDefault();
      if (!user) {
        return;
     }


```

Reply:
One other thing I thought of:
When are we naming an object `user` and when do we name it `patron`?